apiary.apib

```diff
@@ -2474,6 +2474,22 @@ Create a new project.
         + title: `New company website` (string, required)
         + description (string, optional)
         + starts_on: `2016-02-04` (string, required)
+        + billing_method (object, optional)
```

Comment:
Sorry for bringing up this discussion again, but to me `billing_method` sounds like it has something to do with visa, sepa or PayPal. Isn't there a better domain term for this?